coriolisclient/cli/endpoint_storage.py

```diff
@@ -27,12 +27,11 @@ class EndpointStorageFormatter(formatter.EntityFormatter):
                "Name")
 
     def _get_sorted_list(self, obj_list):
-        return sorted(obj_list, key=lambda o: o.name)
+        return sorted(obj_list, key=lambda o: o['name'])
```

Comment:
This is making me uneasy (even knowing Coriolis validates all returned values are dicts have a `name` field), could we do an `o.get('name')` just to be safe?